frontend/src/tests/ddah-tests.js

```diff
@@ -272,20 +272,21 @@ export function ddahTests(api) {
         expect(resp).toHaveStatus("success");
         expect(resp.payload.approved_date).toBeFalsy();
 
-        // Updating a signed DDAH should remove the signature
-        resp = await apiPOST(`/admin/ddahs`, ddah);
+        // Sign the DDAH
+        resp = await apiPOST(
+            `/public/ddahs/${ddah.url_token}/accept`,
+            { signature: "My Sig" },
+            true
+        );
```

Comment:
Signing the DDAHs should be done through a special route, and not through POSTing to the `admin/ddahs` route.